core/src/test/scala/tech/beshu/ror/accesscontrol/blocks/definitions/ldap/implementations/ReleasablePoolTest.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.accesscontrol.blocks.definitions.ldap.implementations
+
+import cats.{Id, Monad}
+import monix.execution.atomic.{AtomicBoolean, AtomicInt}
+import org.scalatest.WordSpec
+import tech.beshu.ror.accesscontrol.blocks.definitions.ldap.implementations.ReleasablePoolTest.Counter
+import cats.implicits._
+import monix.eval.Task
+import monix.execution.schedulers.TestScheduler
+
+import concurrent.duration._
+import language.postfixOps
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5113380)